app/components/VerticalSwipeView/VerticalSwipeView.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ImageBackground, ScrollView, View } from 'react-native';
+
+import { generalStyles } from '../../utils/styles';
+import { ExtendedButton } from '../ExtendedButton';
+import { IVerticalSwipeViewProps } from './VerticalSwipeView.props';
+import { styles } from './VerticalSwipeView.styles';
+
+export const VerticalSwipeView: React.FC<IVerticalSwipeViewProps> = ({
+  renderTopElement,
+  renderBottomElement,
+  backgroundImage,
+}) => {
+  const { t } = useTranslation();
+
+  const scrollViewRef = useRef<ScrollView>(null);
+
+  const onDownPress = useCallback(() => {
+    scrollViewRef.current?.scrollToEnd();
+  }, []);
+
+  const onUpPress = useCallback(() => {
+    scrollViewRef.current?.scrollTo({ x: 0, y: 0, animated: true });
+  }, []);
+
+  return (
+    <View style={generalStyles.flex}>
+      <ImageBackground source={backgroundImage} style={generalStyles.flex}>
+        <ScrollView
+          scrollEnabled={false}
+          ref={scrollViewRef}
+          style={styles.scrollView}>
+          <View style={styles.topContentContainer}>
+            {renderTopElement && renderTopElement()}
```

Comment:
We should support swipe logic, please take a look for it with `react-native-gesture-handler` GestureDetector:
https://docs.swmansion.com/react-native-gesture-handler/docs/gesture-composition

Reply:
not now